selector.js

```diff
@@ -0,0 +1,37 @@
+export function customQuerySelector(root, selector) {
+  const nodeStack = [root];
+
+  while (nodeStack.length > 0) {
+    const node = nodeStack.pop();
+
+    if (node.matches(selector)) {
+      return node;
+    }
+    if (node.children) {
+      for (let i = 0; i < node.children.length; i++) {
+        nodeStack.push(node.children[i]);
+      }
```

Comment:
배열을 모두 순회하는 거라면 for문보다는 forEach를 사용했을 때 명확하게 이해가 가능합니다 :)
```suggestion
      node.children.forEach(child => nodeStack.push(child))
```

Reply:
forEach를 사용해서 더 간결하게 바꿀 수 있네요..!